src/cljs/org/broadinstitute/firecloud_ui/page/workspace/method_configs/tab.cljs

```diff
@@ -70,26 +71,19 @@
    :component-did-mount
    (fn [{:keys [this]}]
      (react/call :load this))
-   :component-did-update
-   (fn [{:keys [this state]}]
-     (when (nil? (:server-response @state))
-       (react/call :load this)))
```

Comment:
Not sure why this was here, it might have been a holdover from the old load-counter implementation (see deletions below).  This was causing an infinite loop of loading, and removing it doesn't seem to break anything.